### PR TITLE
 Include font size property for text tool #444

### PR DESCRIPTION
**Pull request by M06631**
# Feature

### CODE REVIEW

**Comment by M06631** [comment]
@satu0king
yes it persist the size to database
also I refactored it using mutableProperties (BTW it was a great extending feature)

**Comment by M06631** [comment]
@fluctix Sure
Screenshot

**Comment by fluctix** [comment]
LGTM👍

File: public/js/modules.js [hunk 1]
```diff
@@ -1083,15 +1104,18 @@ Text.prototype.draw = function () {
     if ((this.hover && !simulationArea.shiftDown) || simulationArea.lastSelected == this || simulationArea.multipleObjectSelections.contains(this)) {
         ctx.beginPath();
         ctx.fillStyle = "white";
-        rect2(ctx, -this.leftDimensionX, -this.upDimensionY, this.leftDimensionX + this.rightDimensionX, this.upDimensionY + this.downDimensionY, this.x, this.y, "RIGHT");
+        var magicDimenstion = this.fontSize - 14;
```
Comment (satu0king) [inline public/js/modules.js]: Nice labelling but I think this is a typo.

**Comment by satu0king** [comment]
Good Job!

**Pull request merged**